### PR TITLE
Piles and storage pit

### PR DESCRIPTION
**Pull request by bwrsandman**
This PR does three things:
* Decodes the Extra Metrics L3D block.
* Implements the `CREATE_POT` script command
* Adds the Wood and Food piles to storage pits

image
image
image

### CODE REVIEW

**Comment by bwrsandman** [comment]
The fail on x86 is unrelated and happens on every test. I suspect a bad cache.

File: src/ECS/Components/StoragePit.h [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#include <array>
+
+#include <entt/fwd.hpp>
+
+namespace openblack::ecs::components
+{
+
+struct StoragePit
+{
+	std::array<entt::entity, 5> woodPiles;
```
Comment (raffclar) [inline src/ECS/Components/StoragePit.h:21]: Where did the 5 come from?

**Review by raffclar — COMMENTED** [review]
Really good. Just 1 q about the constant in one of the components

**Review by raffclar — APPROVED** [review]

File: src/ECS/Components/StoragePit.h (hunk 1 above)
Comment (bwrsandman) [reply src/ECS/Components/StoragePit.h:21]: There are 5 separate piles here
image

Comment (raffclar) [reply src/ECS/Components/StoragePit.h:21]: Ah ok got it

**Pull request merged**